Common/src/Storage/Result.cs

```diff
@@ -28,24 +28,22 @@ namespace ArmoniK.Core.Common.Storage;
 /// <summary>
 /// </summary>
 /// <param name="SessionId">Id of the session that produces and consumes this data</param>
+/// <param name="ResultId">Unique Id of the result</param>
 /// <param name="Name">Name to reference and access this result</param>
 /// <param name="OwnerTaskId">Id of the task that is responsible of generating this result.</param>
 /// <param name="Status">Status of the result (can be Created, Completed or Aborted)</param>
 /// <param name="DependentTasks">List of tasks that depend on this result.</param>
 /// <param name="CreationDate">Date of creation of the current object.</param>
 /// <param name="Data">Data for the current <paramref name="Name" /></param>
 public record Result(string       SessionId,
+                     string       ResultId,
```

Comment:
Why is the resultId not the first field?

Reply:
SessionId is also the first field in TaskData. I did the same here